letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -110,7 +125,7 @@ su_sudo() {
 
 SUDO_ENV=""
 export CERTBOT_AUTO="$0"
-if [ -n "${LE_AUTO_SUDO:+x}" ]; then
```

Comment:
Indeed, I think I overlooked this detail in @bmw's code review and accidentally wrote it as `${LE_AUTO_SUDO:+x}`, since I wasn't aware of the version without colon to test for only unset variables.